src/View/HelperRegistry.php

```diff
@@ -66,8 +66,9 @@ public function __isset($helper)
         try {
             $this->load($helper);
         } catch (MissingHelperException $exception) {
-            if ($this->_View->getPlugin()) {
-                $this->load($this->_View->getPlugin() . '.' . $helper);
+            $plugin = $this->_View->getPlugin();
+            if (!empty($plugin)) {
```

Comment:
Could also be now `!== null` right?

Reply:
Same reason as above.

Reply:
Thats my point: It can only ever be a non-empty string or null. If it can also contain empty string, then `!$plugin` would be the safer bet.